### ISSUE TITLE
AWD Gpus

### ISSUE THREAD

**Issue by FabianSchuetze** [cloud-gpus/cloud-gpus.github.io#33]
Are there any providers giving access to AMD Gpus? Could they be added to the list?

**Comment by hcarlens** [comment]
Yes, these will be added soon. Feel free to add to this thread if you know any I should add.
